packages/dds/tree/src/test/simple-tree/simpleTree.bench.ts

```diff
@@ -80,6 +82,266 @@ describe("SimpleTree benchmarks", () => {
 				},
 			});
 		}
+
+		function testAccessToLeaf<RootSchema>(
```

Comment:
Looking at the usage of this type parameter, it seems to be the type of a node, not the type of a schema. Thus I think calling it "Root" or "RootNode" would be easier to understand.

Reply:
Renamed to `RootNode`